apps/ledger-live-desktop/src/newArch/WalletSync/SideContent/router.tsx

```diff
@@ -1,12 +1,25 @@
-import React from "react";
+import React, { forwardRef } from "react";
 import { useSelector } from "react-redux";
 import { Flow, walletSyncFlowSelector, walletSyncSelector } from "~/renderer/reducers/walletSync";
 import WalletSyncActivation from "LLD/WalletSync/SideContent/Activation";
 import WalletSyncManage from "LLD/WalletSync/SideContent/Manage";
 import { Flex, InfiniteLoader } from "@ledgerhq/react-ui";
 import Synch from "./Synch";
+import WalletSyncManageBackups from "./ManageBackup";
 
-export const WalletSyncRouter = () => {
+export const STEPS_WITH_BACK: Record<Flow, number[]> = {
+  [Flow.ManageBackups]: [1, 2],
+  [Flow.Activation]: [],
+  [Flow.Synchronize]: [],
+  [Flow.ManageInstances]: [],
```

Comment:
We already define this logic in useFlows hook. We should have only one source of truth to configure the steps and associated configuration

Reply:
It's not the same, Has we have only few screens with a "back" button, It's not to handle next/previous steps